Mathlib/GroupTheory/FreeAbelianGroup.lean

```diff
@@ -193,12 +193,17 @@ theorem lift.add' {α β} [AddCommGroup β] (a : FreeAbelianGroup α) (f g : α
   · intro x y hx hy
     simp only [(lift _).map_add, hx, hy, add_add_add_comm]
 
+/-- `FreeAbelianGroup.lift` as an equivalence of groups. -/
+@[simps!]
+def liftAddEquiv {α G : Type*} [AddCommGroup G] : (α → G) ≃+ (FreeAbelianGroup α →+ G) :=
+  ⟨lift, fun _ _ ↦ by ext; simp⟩
```

Comment:
I think this proof should be called `lift_add'`, to match `lift_neg'` below.

Reply:
Did you comment on the wrong range? Why a prime?